### PR TITLE
Add project tremor assessment

### LINKED ISSUE #47: Project Tremor docs assessment

### PR DESCRIPTION
**Pull request by celestehorgan**
Signed-off-by: Celeste Horgan <celeste@cncf.io>

Closes #47

### CODE REVIEW

**Comment by nate-double-u** [comment]
Contributes to #32

File: assessments/0003-tremor.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+## Recommendations
+
+**Reorganize your documentation's information architecture:** In addition to the suggestions above, I recommend an overall reorganization of the documentation, along the lines of: 
+
+- Docs
+    - Overview (renamed from Architecture Overview)
+    - Using Tremor 
+        - .. subpages as described previously 
+    - Artefacts
```
Comment (nate-double-u) [inline assessments/0003-tremor.md:194]:
```suggestion
    - Artifacts 
```

(I like the "artefact" variant too, but I don't think it's widely used any more)

**Review by nate-double-u — APPROVED** [review]
Thanks @celestehorgan, this is a good assessment. I've made some comments in line regarding grammar, structure, and formatting, but they're minor. The docs review itself is thorough and helpful.

File: assessments/0003-tremor.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        - [The Configuration walkthrough](https://docs.tremor.rs/operations/configuration-walkthrough/) was the single most useful page in terms of understanding the overall 'picture' of what Tremor is and how a developer needs to move through the documentation. But it's buried in the Configuration section, and I wouldn't think to look there until I was much further in the process.
+        - Given how valuable the [Workshops](https://docs.tremor.rs/workshop/examples/00_passthrough/) section is, it needs a better name. "Recipes", "Tutorials", "How to" would all be clearer. 
+    - Some specific comments about the Tremor Query and Script sections:
+        - The [Tremor Query Overview](https://docs.tremor.rs/tremor-query/) and [Tremor Script Overview](https://docs.tremor.rs/tremor-script/) would benefit from splitting the "Language" topic and subtopics into its own page in the section.
```
[the remark this answers is not in the document]
Comment (celestehorgan) [reply assessments/0003-tremor.md:47]: Good catch! The correct wording is actually: 
```suggestion
        - The [Tremor Query Overview](https://docs.tremor.rs/tremor-query/) and [Tremor Script Overview](https://docs.tremor.rs/tremor-script/) would benefit from splitting the "Language" topic and subtopics into their own page in the section. 
```

File: assessments/0003-tremor.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    - I like the [FAQ](https://www.tremor.rs/faq/) as new user content, but FAQs often become a catch-all and get messy quickly. Proceed with caution.
+
+- **Content maintainability:**  
+    - Docs are mostly separated into their own repository, https://github.com/tremor-rs/tremor-www-docs, though as the maintainers note there are many other repositories with docs available in them. These need to be consolidated for easier maitenence. 
+    - The [scripts](https://github.com/tremor-rs/tremor-www-docs/tree/main/python_scripts) which generate certain pieces of documentation introduce fragility into the system: if the location of files, names of repos, etc. change then the documentation pipeline breaks. This also makes it hard for non-"technical" users to make changes to these pages, and to reorganize these pages if needed.
+        - The automation benefits might be worth it however. 
+    - Your [release process](https://github.com/tremor-rs/tremor-www-docs/blob/main/RELEASE_PROCESS.md) is documented, which is great to see!
+    - Your site is also searchable!
```
[the remark this answers is not in the document]
Comment (celestehorgan) [reply assessments/0003-tremor.md:70]: Good catch!

**Comment by celestehorgan** [comment]
Dang, this branch is messed up :( merge it and hope for the best?

File: howto/website-guidelines-checklist.md [hunk 4]
```diff
@@ -5,8 +5,6 @@ As per the [CNCF Website Guidelines](https://github.com/cncf/foundation/blob/mas
 
 - [ ] 1. Website should be [hosted in an open source repo](./repo-setup.md)
   - [ ] Hosted in the same organization as the main project
-  - [ ] Setup [DCO](https://github.com/apps/dco) or CLA (DCO recommended)  
-   CNCF's IP policy (https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy) requires all projects to use either CLA (Contributor License Agreements) or [DCO (Developer Certificate of Origin)](https://github.com/apps/dco). Unless there's a strong necessity to use CLA, we encourage projects to use DCO as it's easier to setup and use.
```
Comment (nate-double-u) [inline howto/website-guidelines-checklist.md:9]: We may want to remove these points, but I'm not sure this is the PR to do that in.

**Review by nate-double-u — CHANGES_REQUESTED** [review]
I'm not sure what's happened with the PR, but now it looks like we're editing more files than we intended to.

File: howto/website-guidelines-checklist.md (hunk 4 above)
Comment (celestehorgan) [reply howto/website-guidelines-checklist.md]: I think I resolved this by dropping the commit, but I'm gonna take a backup of the file just in case.

**Pull request merged**